From 9161ea5aae4354e39088e73824cbbde8b362dfc7 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Mon, 14 Sep 2026 21:34:49 +0800
Subject: [PATCH] AOSCOS: RETRO: fix(kicker): do not show entry to the full
 control center

The Trinity Control Center renders HTMLs as its index pages, causing RAM
drain. We do not have much to use out of the 128MiB budget.

In the meanwhile, the KCM entries are still accessible as individual
modules and they are much, much lighter.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 kicker/menuext/prefmenu/prefmenu.cpp | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/kicker/menuext/prefmenu/prefmenu.cpp b/kicker/menuext/prefmenu/prefmenu.cpp
index 5440c1ed7..79f0bb695 100644
--- a/kicker/menuext/prefmenu/prefmenu.cpp
+++ b/kicker/menuext/prefmenu/prefmenu.cpp
@@ -242,14 +242,6 @@ void PrefMenu::initialize()
     // you click to open the menu, and then press and drag an item.
     m_dragStartPos = TQPoint(-1,-1);
 
-    if (m_root.isEmpty())
-    {
-        insertItem(KickerLib::menuIconSet("kcontrol"),
-                   i18n("Trinity Control Center"),
-                   this, TQ_SLOT(launchControlCenter()));
-        insertSeparator();
-    }
-
     // We ask KSycoca to give us all services under Settings/
     KServiceGroup::Ptr root = KServiceGroup::group(m_root.isEmpty() ? "Settings/" : m_root);
 
-- 
2.55.0

